roles/openshift_logging_mux/defaults/main.yml

```diff
@@ -1,7 +1,7 @@
 ---
 ### General logging settings
-openshift_logging_mux_image_prefix: "{{ openshift_hosted_logging_deployer_prefix | default('docker.io/openshift/origin-') }}"
-openshift_logging_mux_image_version: "{{ openshift_hosted_logging_deployer_version | default('latest') }}"
+openshift_logging_mux_image_prefix: "{{ openshift_logging_image_prefix | default(__openshift_logging_image_prefix) }}"
```

Comment:
As I said "If we don't feel that is important, then I'm fine with getting rid of the separate mux prefix/version parameters." Sounds like it is less important than the testing matrix consideration, so I'm ok with having one version of fluentd for both fluentd and mux.